pravega/pravega
# Issue 1239: Pathing jar to provide the classpath when running on windows

**Change log description**
Relates to issue #1239.  On windows the classpath generated in the application scripts is too long.  This creates an extra JAR that is used purely for providing the classpath to start up the application.  This was adapted from an example on the gradle issue (https://issues.gradle.org/browse/GRADLE-2992)

**Purpose of the change**
Provide the ability to run the pravega applications on windows.

**What the code does**
Creates an empty JAR with a manifest providing the classpath to use for actually running the application.

**How to verify it**
Extract (or build) the distribution on windows and see that the application starts up without the `The input line is too long.` error.
